index.html

```diff
@@ -1,51 +1,92 @@
-                <p id="date">20.02.2020</p>
-            </article>
-            <article class="article">
-                <h2>my second article</h2>
-                <p id="text">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec id nibh sit amet tortor rhoncus pharetra non fermentum quam. Nam dui libero, ultricies a mi vitae, convallis congue felis. Aliquam erat volutpat. In sit amet faucibus tellus, vel semper justo. Fusce ullamcorper pharetra iaculis. Donec venenatis porta dolor, id viverra.</p>
-                <p id="date">24.04.2020</p>
-            </article>
-        </div>
-        <hr>
-    </main>
-    <footer>  
-        <p>Coded by: Wiktor Bonderek</p>
-        <p>wiktor.bonderek@shoper.pl</p>
-    </footer>
-</body>
+ <head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <!-- <link rel="stylesheet" href="reset.css"> -->
+  <link rel="stylesheet" href="style.css">
+  <title>Lista zakupów</title>
+ </head>
+ <body>
+  
+   <header>
+     <hr>
+     <h1>Lista zakupów</h1>
+     <span>
+      <hr>
+      <hr>
+    </span>
+   </header>
+
+   <main>
+
+    <div class="wrapper">
+
+      <div class="fruits">
+        <div class="hero">
+          <h2>OWOCE</h2>
+        </div>
+        <div class="text-content">
+          <ul>
+            <li>Jabłka</li>
+            <li>Pomarańcze</li>
+            <li>Kiwi</li>
+          </ul>
+        </div>
+        <div class="checkbox">
+          <label><input type="checkbox" id="">Owoce kupione</label>
+        </div>
+      </div>
+
+      <div class="veggies">
+        <div class="hero">
+          <h2>WARZYWA</h2>
+        </div>
+        <div class="text-content">
+          <ul class="list">
+            <li>Sałata</li>
+            <li>Ogórki
+              <ul>
+                <li class="test">Kiszone</li>
+                <li class="test">Świeże</li>
+              </ul>
+            </li>
+            <li>Marchew</li>
+          </ul>
+        </div>
+        <div class="checkbox">
+          <label><input type="checkbox" id="">Warzywa kupione</label>
+        </div>
+      </div>
+
+      <div class="dairy">
+        <div class="hero">
+          <H2>NABIAŁ</H2>
+        </div>
+        <div class="text-content">
+          <ul>
+            <li>Mleko</li>
+            <li>Twaróg</li>
+            <li>Sery
+              <ul>
+                <li>Żółty</li>
+                <li>Camembert</li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+        <div class="checkbox">
+          <label><input type="checkbox" id="">Nabiał kupiony</label>
```

Comment:
```suggestion
          <label>
                 <input type="checkbox" id="">
                 Nabiał kupiony
          </label>
```
imo takie formatowanie dużo czytelniejsze